uspace/app/bdsh/cmds/modules/ls/ls.c

```diff
@@ -87,14 +95,46 @@ static unsigned int ls_start(ls_job_t *ls)
  *
  * @param de		Directory element.
  */
-static void ls_print(struct dir_elem_t *de)
+static bool ls_print(struct dir_elem_t *de)
```

Comment:
Please don't use `bool` return type as an error indicator.
I know some other code uses this convention, but it tends to obscure meaning, so we should be trying to reduce it.
The two "good" options you have is to either use `errno_t` and return `<errno.h>` constants as appropriate, or to use the `CL_E...` constants resident in bdsh.